src/library/Uno.Material/Styles/Controls/v2/PasswordBox.xaml

```diff
@@ -89,44 +196,40 @@
+										<Setter Target="ContentElement.Foreground" Value="{ThemeResource FilledPasswordBoxForegroundPointerOver}" />
+										<Setter Target="PlaceholderElement.Foreground" Value="{ThemeResource FilledPasswordBoxPlaceholderForegroundPointerOver}" />
+										<Setter Target="Root.Background" Value="{ThemeResource FilledPasswordBoxBackgroundPointerOver}" />
+										<Setter Target="NormalBorder.Height" Value="2" />
 									</VisualState.Setters>
 								</VisualState>
 
 								<VisualState x:Name="Pressed" />
 
 								<VisualState x:Name="Disabled">
 									<VisualState.Setters>
-										<Setter Target="NormalBorder.Fill"
-												Value="{ThemeResource OnSurfaceLowBrush}" />
-										<Setter Target="ContentElement.Opacity"
-												Value="{StaticResource DisabledOpacity}" />
-										<Setter Target="PlaceholderElement.Foreground"
-												Value="{ThemeResource OnSurfaceLowBrush}" />
-										<Setter Target="Root.Background"
-												Value="{ThemeResource OnSurfaceDisabledLowBrush}" />
+										<Setter Target="NormalBorder.Fill" Value="{ThemeResource FilledPasswordBoxBorderBrushDisabled}" />
+										<Setter Target="ContentElement.Foreground" Value="{ThemeResource FilledPasswordBoxForegroundDisabled}" />
+										<Setter Target="PlaceholderElement.Foreground" Value="{ThemeResource FilledPasswordBoxPlaceholderForegroundDisabled}" />
+										<Setter Target="Root.Background" Value="{ThemeResource FilledPasswordBoxBackgroundDisabled}" />
 									</VisualState.Setters>
 								</VisualState>
 
 								<VisualState x:Name="Focused">
 									<VisualState.Setters>
-										<Setter Target="PlaceholderElement.Foreground"
-												Value="{ThemeResource PrimaryBrush}" />
-										<Setter Target="NormalBorder.Fill"
-												Value="{ThemeResource PrimaryBrush}" />
-										<Setter Target="NormalBorder.Height"
-												Value="2" />
+										<Setter Target="NormalBorder.Fill" Value="{ThemeResource FilledPasswordBoxBorderBrushFocused}" />
+										<Setter Target="ContentElement.Foreground" Value="{ThemeResource FilledPasswordBoxForegroundFocused}" />
+										<Setter Target="PlaceholderElement.Foreground" Value="{ThemeResource FilledPasswordBoxPlaceholderForegroundFocused}" />
+										<Setter Target="Root.Background" Value="{ThemeResource FilledPasswordBoxBackgroundFocused}" />
+										<Setter Target="NormalBorder.Height" Value="2" />
 									</VisualState.Setters>
 
 									<Storyboard>
-										<DoubleAnimation EasingFunction="{StaticResource MaterialEaseInOutFunction}"
+										<DoubleAnimation EasingFunction="{StaticResource EaseInOutFunction}"
 														 Storyboard.TargetName="Scale"
 														 Storyboard.TargetProperty="ScaleX"
 														 To="1"
-														 Duration="{StaticResource MaterialTextBoxAnimationDuration}" />
+														 Duration="{StaticResource TextBoxAnimationDuration}" />
```

Comment:
Should this still have the Material prefix on it?

Reply:
I had missed to revert this, it's done